### ISSUE TITLE
Limiting the rate of consuming

### ISSUE THREAD

**Issue by rbozan** [rusty-celery/rusty-celery#370]
Hi, more a question than a request but it's not clear tp me how I would limit the rate of consumption? It seems that all my tasks are now instantly being consumed by the consumer instead of limiting it to e.g. 3 at a time.

**Comment by epwalsh** [comment]
Hey @rbozan it sounds like you're looking for the [`prefetch_count`](https://docs.rs/celery/latest/celery/struct.CeleryBuilder.html#method.prefetch_count) setting. For both the AMQP and Redis brokers this defaults to 10.

**Comment by rbozan** [comment]
I have tried that and actually have set it to `1` but that does not seem to work how I want it to..  For example, how would I limit a certain task to handling only 1 at a time and another task to handle 5 at a time?

**Comment by epwalsh** [comment]
> how would I limit a certain task to handling only 1 at a time and another task to handle 5 at a time?

I don't think that's possible ATM with a single consumer. But in that scenario I would consider using a separate queue and consumer for those different tasks.

**Comment by rbozan** [comment]
Okay but still, I have set `prefetch_count = 1` but it still seems to handle multiple at a time.  It doesn't wait until one task finished before starting the next.

EDIT: My fault, I needed `acks_late = true`. Thanks!
